scripts/compose.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Script based on https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/scripts/compose.js
+ */
+
+const fs = require('fs')
+const inquirer = require('inquirer')
+const dedent = require('dedent')
+
+const genFrontMatter = (answers) => {
+  let d = new Date()
+  const date = [
+    d.getFullYear(),
+    ('0' + (d.getMonth() + 1)).slice(-2),
+    ('0' + d.getDate()).slice(-2),
+  ].join('-')
+  const tagArray = answers.tags.split(',')
+  tagArray.forEach((tag, index) => (tagArray[index] = tag.trim()))
+  const tags = "'" + tagArray.join("','") + "'"
+
+  let frontMatter = dedent`---
+  title: ${answers.title ? answers.title : 'Untitled'}
+  date: '${date}'
+  type: ${answers.type}
+  tags: [${answers.tags ? tags : ''}]
+  cover: /img/posts/may-2021-at-asyncapi/cover.webp
+  draft: ${answers.draft === 'yes' ? true : false}
+  authors:
+    - name: Lukasz Gornicki
+      photo: /img/avatars/lpgornicki.webp
+      link: https://twitter.com/derberq
+      byline: AsyncAPI Maintainer and Community Guardian
+  excerpt: ${answers.excerpt ? answers.excerpt : ' '}
+  ---
+
+  Write your blog post content here, just remember to mention "AsyncAPI" :smile:. If you need a refresher on Markdown,
+  you can take a look at [this guide](https://tailwind-nextjs-starter-blog.vercel.app/blog/github-markdown-guide).
+
+  ## Test sub-section 1
+
+  **Authors**
+
+  Before submitting your blog post, don't forget to change the \`authors\` array field to include yourself :smile:
+
+  ## Test sub-section 2
+
+  **Images**
+
+  Please make sure to give credit to source of the image, for example:
+
+  > Cover image by <a href="https://pixabay.com/users/silviarita-3142410/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=2634391">silviarita</a> from <a href="https://pixabay.com/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=2634391">Pixabay</a>
+
+  Add a cover image, you can change it on the metadata field above called \`cover\`. In case you need some inspiration we recommend https://unsplash.com/.
+  All images should be stored in the \`public/img/posts/\` folder, below is an example of using an image in your post:
+
+  ![Figure 3: APIs give developer teams more independence](/img/posts/openapi-vs-asyncapi-burning-questions/asyncapi-openapi-post_pic-03.webp)
```

Comment:
I would recommend using dedicated component:
```
<Figure
  src="/img/posts/2020-summary/linkedin-folowers.webp"
  caption="Figure 3: Followers growth in 2020"
/>
```

as it also enforces you think about accessibility because caption will also be used as alt text. You can also mention it